assets/css/custom.css

```diff
@@ -1896,4 +1905,566 @@ a.read-all-posts:hover {
+}
+
+.new-header .main_menu>li.another_link a[target=_blank] {
+    padding-right: 1.7rem;
+    position: relative;
+}
+
+.new-header .main_menu>li.another_link a[target=_blank]:before {
+  background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg width='10' height='11' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M9.536 0H6.338a.464.464 0 1 0 0 .928h2.129L3.911 5.484a.464.464 0 1 0 .657.656l4.504-4.504v2.026a.464.464 0 1 0 .928 0V.464A.464.464 0 0 0 9.536 0Z' fill='%234720E3'/%3E%3Cpath d='M7.93 4.675a.464.464 0 0 0-.464.465v3.662c0 .168-.142.31-.31.31H1.238a.314.314 0 0 1-.31-.31V2.883c0-.167.142-.31.31-.31h3.675a.464.464 0 1 0 0-.927H1.238A1.24 1.24 0 0 0 0 2.883v5.919a1.24 1.24 0 0 0 1.238 1.238h5.918a1.24 1.24 0 0 0 1.238-1.238V5.14a.464.464 0 0 0-.464-.465Z' fill='%234720E3'/%3E%3C/svg%3E");
+  background-position: 50%;
+  background-repeat: no-repeat;
+  background-size: contain;
+  content: "";
+  height: 1.2rem;
+  position: absolute;
+  right: 0;
+  top: 0;
+  transition: .25s;
+  width: 1rem;
+}
+
+@media (min-width:1200px) {
+  .new-header .main_menu>li.another_link a[target=_blank]:before {
+    height: 1.4rem;
+    width: 1.25rem;
+  }
+}
+
+.new-header .links_btns {
+  margin-left: 1.5rem;
+  padding-bottom: 26px;
+  transition: .25s;
+}
+
+@media (max-width:880px) {
+  .new-header .links_btns {
+    display: none;
+  }
+}
+
+.new-header .links_btns a {
+  display: inline-block;
+  width: 2rem;
+}
+@media (min-width:1200px) {
+  .new-header .links_btns a {
+    width: 2.7rem;
+  }
+}
+.new-header .links_btns a:first-of-type {
+  margin-right: 0.6875rem;
+}
+
+.new-header .links_btns a svg {
+  width: 100%;
+}
+.new-header .links_btns a:hover svg path {
+  fill: #0dddd3;
+}
+
+.new-header .links_btns a svg path {
+  transition: fill .25s;
+  width: 100%;
+}
+
+
+/* Mobile Nav */
+.new-header .mobile_menu {
+  background-color: #fff;
+  box-shadow: 0.25rem 0.25rem 3.125rem rgba(0,0,0,.25);
+  display: none;
+  height: 100vh;
+  max-width: 24rem;
+  opacity: 0;
+  overflow: auto;
+  padding: 1rem 1rem 5rem;
+  position: absolute;
+  right: 0;
+  top: 78px;
+  transform: translateX(100%),scaleX(0);
+  transition: transform .4s cubic-bezier(.77,.2,.05,1),opacity .4s ease,visibility .4s ease;
+  visibility: hidden;
+  width: 80vw;
+  z-index: 15;
+
+  opacity: 1;
+  transform: translateX(0),scaleX(1);
+  visibility: visible;
+  padding: 1.8rem 1.8rem 8rem;
+}
+
+@media (max-width:880px) {
+  .new-header .mobile_menu {
+    display: none;
+  }
+  .new-header .mobile_menu.is_active {
+    display: block;
+  }
+}
+
+.new-header .mobile_menu .main_menu {
+  opacity: 1;
+  transform: translateX(0),scaleX(1);
+  visibility: visible;
+  display: block;
+}
+
+.new-header .mobile_menu .main_menu>li  {
+  display: block;
+}
+
+.new-header .mobile_menu .main_menu>li>button {
+  border-bottom: none;
```

Comment:
```suggestion
```

On the mobile nav, the menu items jump around as you hover over them. This is the first of three changes to fix that.